torch/linalg/__init__.py

```diff
@@ -419,12 +452,12 @@
              in the real case or by :math:`e^{i \phi}` for any :math:`\phi \in \mathbb{R}` in the
              complex case, and the resulting singular vectors will give a different
              eigendecomposition of the matrix.
-             As such, the eigenvectors are not continuous functions of the inputs, and different
-             platforms, like NumPy, or inputs on different devices, may produce different
-             eigenvectors.
+             As this choice is not canonical, it happens that the eigenvectors are not continuous
+             functions of the input matrix. As such, different platforms, like NumPy, or inputs
+             on different devices, may produce different eigenvectors.
 
-.. warning:: If `V` is used to compute gradients, the gradient will only be finite when
-             :attr:`input` does not have repeated eigenvalues.
+.. warning:: If the `eigenvectors` are used to compute gradients, the gradient will only be
```

Comment:
"Gradients computed using the \`eigenvectors\` tensor will only be finite when :attr:\`A\` has unique eigenvalues."

Reply:
Corrected here and in the `svd`. In my opinion the wording is a bit obscure, but I cannot suggest at the moment a better one.